documentation/kernel-semantics.tex

```diff
@@ -1576,30 +1556,35 @@ \subsubsection{Function Invocation}
             \cesktranswheresplit%
                 {\evalconf{\StaticGet{\membermeta}}{\env}{\strace}{\cstrace}{\cex}{\econt}}%
                 {\contconf{\econt}{\deref{\mainenv(\membermeta)}}}%
-                {\text{where $\membermeta$ is a static field and $\membermeta \in \dom(\mainenv$)}}
+                {\text{where $\membermeta$ is a static field and $\deref{\mainenv(\membermeta)} \neq \emptyset$}}
             \end{multlined}
         \label{eval:staticget-var}\\
         &\begin{multlined}
             \cesktranswheresplit%
                 {\evalconf{\StaticGet{\membermeta}}{\env}{\strace}{\cstrace}{\cex}{\econt}}%
                 {\contconf{\econt}{\nnull}}%
-                {\parbox{12cm}{
-                    where $\membermeta$ is a static field without an initializer expression and $\membermeta \notin \mainenv$,\\
-                    $\mainenv = \ext{\mainenv}{\membermeta}{\NullLiteral}$ after transition}}
+                {\begin{aligned}
+                    \text{where } &\membermeta \text{ is a static field without an initializer expression},\\
+                                  &\deref{\mainenv(\membermeta) = \emptyset},\\
+                                  &\update{\mainenv(\membermeta)}{\NullLiteral}\text{ after transition}
```

Comment:
If we go with the co-domain of the global environment being expressed as Empty + Location, then the update should be written in a different way, for example, like this:

```
...
&\mainenv(\membermeta) = \inr(\alpha),\\
&\update{\alpha}{\NullLiteral}\text{ after transition}
...
```

Changes like that should be made here and below.